semcore/utils/src/enhances/WithI18n.tsx

```diff
@@ -67,5 +67,18 @@ const useI18n = (
   );
 };
 
+export const i18nAppLocaleEnhance = () => {
```

Comment:
Can we move this function to the core?
It seems that this api(wrapperProps) is part of the core